server/ops/docker/jdk11-jammy/Dockerfile

```diff
@@ -155,6 +155,34 @@ FROM yuzutech/kroki-builder-erd:0.2.1.0 AS kroki-builder-static-erd
 #  --ghc-options="-optc=-static" \
 #  --ghc-options="-w"
 
+## Symbolator
+FROM ubuntu:jammy AS kroki-builder-static-symbolator
+
+# Build from forked source because upstream is broken for the latest python3 versions
+ARG SYMBOLATOR_SOURCE=git+https://github.com/zebreus/symbolator.git@v1.2.1
+
+RUN apt-get update && apt-get install --no-install-recommends --yes \
+    git \
+    pip \
+    python3-dev \
+    patchelf \
+    python3-gi-cairo \
+    python3-gi \
+    build-essential \
+    libpango1.0-dev && \
+    apt-get clean && apt-get autoremove
+
+WORKDIR /build
+
+# Install latest pip and setuptools
+RUN python3 -m pip install --upgrade pip setuptools
+RUN python3 -m pip install --upgrade nuitka
+# Install symbolator
+RUN python3 -m pip install --upgrade ${SYMBOLATOR_SOURCE}
+
+# Use nuitka to compile a static binary so we dont need python in the final image
+RUN python3 -m nuitka --onefile `which symbolator` --include-module=gi.overrides.Pango --include-module=gi._gi_cairo
```

Comment:
👍🏻

Reply:
> Use nuitka to compile a static binary so we dont need python in the final image

@zebreus Do you think it would be possible to do the same with blockdiag (and wireviz)? My initial goal was to create a binary version but I gave up because I didn't find a working solution. If you think it would be possible, I can create an issue to track progress. Thanks!

Reply:
It seems like nuitka is able to build working static binaries for wireviz without problems. Blockdiag also builds fine but fails at runtime, because it has some missing dependency but that can probably be fixed.

A tracking issue is probably good for further discussion. Do you think we can integrate blockdiag and wireviz into the main container if we have static binaries?

Nuitka also seems to [support multi-call binaries](https://nuitka.net/doc/user-manual.html#use-case-6-multidist) so in the best-case we could build a single binary for all blockdiag packages. Maybe even for all python diagram generators but I am still undecided whether that is a good idea.

Reply:
> Do you think we can integrate blockdiag and wireviz into the main container if we have static binaries?

That could be an option, yes.

> Nuitka also seems to support multi-call binaries so in the best-case we could build a single binary for all blockdiag packages. Maybe even for all python diagram generators but I am still undecided whether that is a good idea.

One for all blockdiag packages is a good idea!

> A tracking issue is probably good for further discussion.

I will create one or since you've already looked into it feel free to create one.

Reply:
That might be because Blockdiag loads "drawer" at runtime using package discovery: https://github.com/blockdiag/blockdiag/blob/ce37f0c18dc7f88caab3d3b24aac4f3f066b793c/src/blockdiag/imagedraw/__init__.py#L23-L31

But maybe we can explicitly tell Nuitka that these resources must be included. Anyway, let's discuss this further in a dedicated issue. I think we can create two issues one for WireViz and one for Blockdiag.

Reply:
@zebreus thanks again for mentioning Nuitka! It does a really good job. I was able to build a single binary for blockdiag and WireViz. Now, I just need to implement the "read from stdin/output to stdout" feature :)